CMakeLists.txt

```diff
@@ -794,10 +794,6 @@ if(USE_COREML_DELEGATE)
   string(APPEND CMAKE_CXX_FLAGS " -DUSE_COREML_DELEGATE")
 endif()
 
-if(BUILD_CAFFE2)
```

Comment:
Hmm, this is reverting the changes in https://github.com/pytorch/pytorch/pull/89199 . Maybe change the stacked PR to avoid such changes?